### PR TITLE
Adds the PubbyStation Monastery as a purchasable shuttle

### PR DESCRIPTION
**Pull request by Son-of-Space**
This PR is an updated version of #54942 , which was closed due to a content freeze.

## About The Pull Request

The PR you never asked for but definitely wanted.

This PR will add a slightly modified version of Pubbystation's Monastery to the list of available emergency shuttles, becoming the most expensive shuttle to buy at 250 times the crate price (Defaults around 50,000cr)

This shuttle will cause significant damage and fully showcase the absurdity of the Monastery's size by providing an up close and personal, interactive comparison. PubbyStation's monastery will be forever preserved.

I have successfully duplicated the method that the meteor shuttle uses to crash into the station and magnified it.

"Today we asketh, Lord, that our prayers run not only our dreams, but our engines."

## Why It's Good For The Game

This shuttle is designed to be one that multiple people have to make a directed effort and put multiple hours of work into purchasing. It's absurd, incredible, and certainly deserving for a shift that has had nothing better to do than farm credits for 3 hours.

This is another one of the crazy shuttles that only madmen (or the clown) purchases, and will probably be converted to an emag-only shuttle when mothblocks makes his PR for it, making it a fun and devastating purchase for traitors or others looking to sabotage the station and possibly hijack.

I also hope that this will preserve in memory the greatness of PubbyStation (Rest in Peace).

## Changelog
:cl:
add: The Pubbystation Monastery is now available as an emergency shuttle.
/:cl:

### CODE REVIEW

**Comment by ShibaInuLord** [comment]
based

**Comment by coiax** [comment]
The notion of stations being removed, but their shuttles staying on is a good tradition.
This is a NEW tradition of cutting off part of the station and making it a shuttle, it's hilarious. 👍 

I look forward to "BirdBoat-but-it's-an-emergency-shuttle".

**Comment by Cartographer-D** [comment]
The monastery was basically it's own mini station, so I thought it apt.

**Review by Cyberboss — APPROVED** [review]

**Comment by Jared-Fogle** [comment]
> This is another one of the crazy shuttles that only madmen (or the clown) purchases, and will probably be converted to an emag-only shuttle when mothblocks makes his PR for it, making it a fun and devastating purchase for traitors or others looking to sabotage the station and possibly hijack.

👍👍

**Review by coiax — CHANGES_REQUESTED** [review]

File: code/game/area/areas/shuttles.dm [hunk 1]
```diff
@@ -163,6 +163,9 @@
 	name = "\proper a meteor with engines strapped to it"
 	luminosity = NONE
 
+/area/shuttle/escape/monastery
+	name = "A magnificent corporate monastery."
```
[the remark this answers is not in the document]
Comment (Cartographer-D) [reply code/game/area/areas/shuttles.dm:167]: Default shuttle area type or /escape/meteor/

[the remark this answers is not in the document]
Comment (Cartographer-D) [reply code/game/area/areas/shuttles.dm:167]: removing the area and replacing it with the default escape shuttle area.

**Comment by cacogen** [comment]
God I can't wait

File: code/modules/admin/fun_balloon.dm [hunk 2]
```diff
@@ -91,16 +91,30 @@
 	icon = 'icons/obj/items_and_weapons.dmi'
 	icon_state = "syndballoon"
 	anchored = TRUE
+	var/shuttle_crash = TRUE //this determines if the shuttle will crash into the station.
```
Comment (coiax) [inline code/modules/admin/fun_balloon.dm:94]: Why have this var at all? Why would you put this effect on a shuttle, only to disable the effect?

File: code/modules/admin/fun_balloon.dm [hunk 3]
```diff
@@ -91,16 +91,30 @@
...
+	var/min_crash_strength = 3
+	var/max_crash_strength = 15
+
+/obj/effect/station_crash/devastating
+	name = "devastating station crash"
+	desc = "Absolute Destruction. Will crash the shuttle far into the station."
+	min_crash_strength = 15
+	max_crash_strength = 25
```
Comment (coiax) [inline code/modules/admin/fun_balloon.dm:102]: Move the subtype definition after all the stuff for the base type.

**Review by coiax — CHANGES_REQUESTED** [review]

**Comment by Son-of-Space** [comment]
I should point out that crashing the shuttle like this does make the game chug quite a bit on local and I don't know how that will reproduce on the servers. Might require a test merge.

**Comment by Son-of-Space** [comment]
We have tested the shuttle on DeltaStation. Here are some highlights:
https://cdn.discordapp.com/attachments/585862469508005888/803709688755650570/unknown.png

**It gibbed a blob when it landed. A+**
MOments before disaster

image

image

image

image

**Comment by Son-of-Space** [comment]
This is ready

**Comment by notfrying1pans** [comment]
beautiful

**Review by coiax — APPROVED** [review]
Knock them dead.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-3). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**